bin/git-coauthor

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
I think we should add `set -eu -o pipefail` at the top of the script. Better to be strict.